GetARock/GetARock/Global/Literal/ImageLiteral.swift

```diff
@@ -14,6 +14,9 @@ enum ImageLiteral {
     static var xmarkSymbol: UIImage { .load(systemName: "xmark") }
     static var arrowUpRightSymbol: UIImage { .load(systemName: "arrow.up.right") }
     static var crownSymbol: UIImage { .load(systemName: "crown") }
+    static var checkMarkInCircle: UIImage { .load(systemName: "checkmark.circle") }
+    static var xMarkInCircle: UIImage { .load(systemName: "x.circle") }
+
```

Comment:
p4;

기존 sf symbol 선언과 비슷하게 네이밍을 변경하면 좋을 것 같아요!
```suggestion
    static var checkmarkCircleSymbol: UIImage { .load(systemName: "checkmark.circle") }
    static var xmarkCircleSymbol: UIImage { .load(systemName: "x.circle") }

```

Reply:
오 감사합니다! 제안주신대로 커밋날렷습니다 ㅎㅎ https://github.com/extreme-rock/GetARock-iOS/pull/77/commits/2bc1704e4f72c780e20935d19967d7d00103db5d